Aboutme.md

```diff
@@ -0,0 +1,19 @@
+### :open_file_folder: What have I learnt?
+* The different principles of Software Development.
+* How to compile a Java
+* The different variables in Java
+
+
+### :confused: What am I confused about?
+* I am not confused about anything, I just need a practical example of SOLID principle of software development. I think once the course progresses I will better understand this. At the moment it's just an abstaract concept.
+
+
+
+### Some of the things that I like
+My life is way too exciting to list all the things that I like not in order of priorty.
```

Comment:
I think you meant to say something like - "the following are not in order of priority"